docs/contributing.md

```diff
@@ -144,6 +144,18 @@ If you are introducing a new feature, you need to either add tests to
 existing classes, or add a new class (if your new feature is
 significantly different from existing code).
 
+#### Old tests and new tests
+Historically, tests were written with unittest and Flask-Testing.
+Because the latter was dying, we removed it and switch as much as
+possible to pytest fixtures. However, a lot of tests were already written,
+and are still using class-based tests, albeit without setUp()/teardown().
+
+If you write new tests, please **try to write them with pytest in mind**:
+- standalone function (not in class)
```

Comment:
Maybe a sentence would be better, like "tests are done in standalone functions rather than in class and class methods".